backend/src/infra/container/infra/repositories.ts

```diff
@@ -1,14 +1,33 @@
+import {
+  EmailRepository,
+  EmailTemplateRepository,
+} from '@/application/repositories';
 import { StudentRepository } from '@/application/repositories/student-repository';
+import { RedisCache } from '@/infra/cache';
+import { EmailCacheRepository } from '@/infra/cache/repositories/email-cache-repository';
 import { prisma } from '@/infra/orm/prisma/datasource';
 import { StudentPrismaRepository } from '@/infra/orm/prisma/repositories';
+import { EmailTemplatePrismaRepository } from '@/infra/orm/prisma/repositories/email-template-prisma-repository';

```

Comment:
Criar um índex para os itens com import grande: 
```suggestion

export * from './nome-do-arquivo';

```